app/models/pipeline_run.rb

```diff
@@ -1038,6 +1057,32 @@ def compile_stats_file
     save
   end
 
+  # Fetch the unmapped reads count from alignment stage then refined counts from
+  # assembly stage, as each becomes available. Prior to Dec 2019, the count was
+  # only fetched from alignment.
```

Comment:
Could you add a comment somewhere about how `all_counts` contains all the `.count` files from `/results`, but we need to fetch this one `.count` file from `/postprocess`? It was a little hard to understand why you needed to fetch another file in this method.